deployment/arm/deploy-global.sh

```diff
@@ -31,14 +44,14 @@ toArmArray() {
   printf $value
 }
 
-let counter=0
+let counter=0 || true
 
 frontendHostArray=()
 backendHostArray=()
 
 for region in $@
 do
-  let counter++
+  let "counter=counter + 1"
```

Comment:
Any reason for this change? I believe it was working before?

Reply:
After setting set -e, the script started to fail on this line, so had to make this change.